Automattic/wp-calypso
# Remove No Ads from the add-on route

Implements the request defined in pau2Xa-4vo-p2 to remove the No-Ads addon.

#### Proposed Changes

* Removes the "No Ads" addon from the `/add-ons` page.
* As an FYI, the parent PR is https://github.com/Automattic/wp-calypso/pull/69462.

**BEFORE**

<img width="1121" alt="Screenshot 2022-10-26 at 1 39 14 PM" src="https://user-images.githubusercontent.com/1269602/197970817-3cc226ea-e758-4cce-8746-73c2ca4051db.png">

**AFTER**

<img width="1435" alt="Screenshot 2022-10-26 at 1 38 53 PM" src="https://user-images.githubusercontent.com/1269602/197970861-37920451-6e21-42b2-bbc5-5edb429190a8.png">


#### Testing Instructions



* Visit http://calypso.localhost:3000/add-ons/{SITE_SLUG} and verify that the "Remove Ads" addon is no longer listed.
